src/AppBundle/Converter/TwigTemplateConverter.php

```diff
@@ -171,7 +175,6 @@ public function convertChildTemplate($mjmlTemplatePath, $mjmlTheme, $newTheme)
 {% endblock %}
 
 {% block styles %}
-{{ parent() }}
```

Comment:
This is a medium change 🤔  is it really needed?

Reply:
This is the intended fix, because `{{ parent() }}` is including the styles of the parent template, and thus all styles get duplicated.

Reply:
Thanks for clarification